mmv1/third_party/terraform/services/compute/resource_compute_instance_template_test.go.tmpl

```diff
@@ -1577,6 +1577,114 @@ func TestAccComputeInstanceTemplate_partnerMetadata(t *testing.T) {
 }
 {{- end }}
 
+func TestAccComputeInstance_IPv6ExternalRecreate(t *testing.T) {
+	t.Parallel()
+
+	var instance compute.Instance
+
+	acctest.VcrTest(t, resource.TestCase{
+		PreCheck:                 func() { acctest.AccTestPreCheck(t) },
+		ProtoV5ProviderFactories: acctest.ProtoV5ProviderFactories(t),
+		CheckDestroy:             testAccCheckComputeInstanceDestroyProducer(t),
+		Steps: []resource.TestStep{
+			{
+				Config: testAccComputeInstanceIPv6External(acctest.RandString(t, 10), false),
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckComputeInstanceExists(
+						t, "google_compute_instance.test", &instance),
+				),
+			},
+			{
+				// troca de IPv6 → recreate obrigatório
+				Config: testAccComputeInstanceIPv6External(acctest.RandString(t, 10), true),
+				Check: resource.ComposeTestCheckFunc(
```

Comment:
Could you possibly include a PlanCheck here as well, to confirm that the update does not create a new instance?